site2/website-next/blog/2019-05-28-hola.md

```diff
@@ -0,0 +1,11 @@
+---
+slug: hola
+title: Hola
+author: Gao Wei
+author_title: Docusaurus Core Team
```

Comment:
what about dropping this pages ?
they refer to unknown folks

Reply:
test md file, this pr is draft, don't do review now